Tribler/Test/Community/Tunnel/test_tunnelcommunity.py

```diff
@@ -120,3 +120,9 @@ def test_send_to_ip_deferred(self):
         data = "ffffffff".decode("HEX") + "1" * 25
         exit_tunnel.sendto(data, ("localhost", -1))
         return exit_tunnel.close()
+
+    def test_increase_bytes_sent_error_branch(self):
```

Comment:
@whirm This only checks if the error is raised, so I think it should not matter?